bridgeActions/actions_automation/retrieve_actions.js

```diff
@@ -0,0 +1,37 @@
+const { Defender } = require('@openzeppelin/defender-sdk');
+
+// Replace with your actual API key and secret
+const creds = { 
+  apiKey: process.env.API_KEY, 
+  apiSecret: process.env.API_SECRET 
```

Comment:
Create an .env.example file where env variables are mentioned without their values and comment of their use case